core/domain/exp_domain_test.py

```diff
@@ -4000,8 +4112,8 @@ class HTMLMigrationUnitTests(test_utils.GenericTestBase):
       customization_args:
         choices:
           value:
-          - <p>This is </p><p>value1 <br>for MultipleChoice</p>
-          - <p>This is value2 for <br>MultipleChoice</p>
+          - <p>This is </p><p>value1 <br/>for MultipleChoice</p>
+          - <p>This is value2 for <br/>MultipleChoice</p>
```

Comment:
Hi @bansalnitish , would it be <br/> or <br> ?

Reply:
And it should be `<br>`. See the convention [here](https://www.w3docs.com/learn-html/html-br-tag.html). Thanks!

Reply:
Hi @ischur7, it should be `<br>`. See my comment here: https://github.com/oppia/oppia/blob/develop/core/domain/html_cleaner.py#L468

Reply:
Thanks @apb7 , @bansalnitish . In the comment mentioned its also written that Beautiful soup automatically changes some `<br>` to `<br/>`. So in any other migration which is run after the conversion of text angular may lead to the conversion of  `<br>` to `<br/>`. I think that is what happened when I tested it on my machine. I have `add_dimensions_to_image` in html_cleaner which adds dimensions to the existing images. The schema version to which it transforms the exp is 30. So I migrated the `YAML_CONTENT_V26_TEXTANGULAR` to schema version 30 and got the assertion error which stated that it should be `<br/>` instead of `<br>`.
Let me know if that's the case. Thanks